client/my-sites/current-site/index.jsx

```diff
@@ -163,15 +167,17 @@ const CurrentSite = React.createClass( {
 // TODO: make this pure when sites can be retrieved from the Redux state
 module.exports = connect(
 	( state ) => {
-		const selectedSiteId = getSelectedSiteId( state );
+		const selectedSiteId = getSelectedSiteId( state ),
+			user = getCurrentUser( state );
 
 		return {
+			isJetpack: isJetpackSite( state, selectedSiteId ),
 			selectedSiteId,
 			selectedSite: getSelectedSite( state ),
-			isJetpack: isJetpackSite( state, selectedSiteId )
+			siteCount: get( user, 'visible_site_count', 0 )
```

Comment:
Hmm, would probably be a good idea! I'd need to refactor quite a few things tho. Maybe do it in a separate PR?